internal/provider/private_endpoint_services_resource_test.go

```diff
@@ -187,7 +187,8 @@ func testPrivateEndpointServicesResource(
 				Config: privateEndpointServicesResourceConfigFn(clusterName),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(clusterResourceName, "name", clusterName),
-					resource.TestCheckResourceAttr("cockroach_private_endpoint_services.services", "services.#", "1"),
+					// TODO: Change the expected number of services to 2 after https://github.com/cockroachlabs/managed-service/pull/13740 is released.
+					resource.TestCheckResourceAttr("cockroach_private_endpoint_services.services", "services.#", "6"),
 					resource.TestCheckResourceAttr("cockroach_private_endpoint_services.services", "services.0.status", string(client.PRIVATEENDPOINTSERVICESTATUSTYPE_AVAILABLE)),
```

Comment:
nit: we might want to loop over the number of expected services to assert that all the services are in the available state.

Reply:
Done!